app/lib/efile/ny/it201.rb

```diff
@@ -430,8 +416,11 @@ def calculate_line_62
       end
 
       def calculate_line_69
-        if line_or_zero(:IT201_LINE_19) < 250_000 && @intake.nyc_full_year_resident_yes?
-          # income calculated as 19a - 9. 9 is not supported and 19a is 19
+        # From IT-201 instructions: Income, for purposes of determining your New York City school tax credit, means your
+        # federal adjusted gross income from Form IT-201, line 19, minus distributions from an individual retirement
+        # account and an individual retirement annuity, from Form IT-201, line 9, if they were included in your federal
+        # adjusted gross income.
+        if (line_or_zero(:IT201_LINE_19) - line_or_zero(:IT201_LINE_9)) < 250_000 && @intake.nyc_full_year_resident_yes?
```

Comment:
I believe we are screening out anyone who is not an nyc_full_year_resident. So there's nothing wrong with this logic but `.nyc_full_year_resident_yes?` should never return false after the eligibility screening pages